codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostStandardRequestDecoder.java

```diff
@@ -486,6 +487,10 @@ private void parseBodyAttributesStandard() {
             // error while decoding
             undecodedChunk.readerIndex(firstpos);
             throw new ErrorDataDecoderException(e);
+        } catch (IllegalArgumentException e) {
```

Comment:
We already do the same when we parse heap buffers but missed it for direct buffers.

Reply:
Not a big deal but why multiple catch blocks here and not just one catching `Throwable`?